src/covid_model_seiir_pipeline/forecasting/task/postprocessing.py

```diff
@@ -0,0 +1,155 @@
+from argparse import ArgumentParser, Namespace
+import functools
+import logging
+import multiprocessing
+from pathlib import Path
+import shlex
+from typing import Dict, Optional, List
+
+import numpy as np
+import pandas as pd
+
+from covid_model_seiir_pipeline import static_vars
+from covid_model_seiir_pipeline.forecasting.specification import ForecastSpecification
+from covid_model_seiir_pipeline.forecasting.data import ForecastDataInterface
+
+log = logging.getLogger(__name__)
+
+LOAD_OUTPUT_CPUS = 3
```

Comment:
Is this a place where you want to scale based off of `fthread`s the job has?

`ihmeutils.cluster.max_cpus()` can do that for you.

Reply:
Oh, I just realized this needs to build outside IHME. Here's the code if it matters - https://stash.ihme.washington.edu/users/miker985/repos/ihmeutils/browse/src/ihmeutils/cluster.py

Reply:
Cool!

I need to come back and properly profile at the end (see todo about reconsidering the usage of multiprocessing).  I wasn't thinking of making this dynamic, but that's a neat package.